openstudiocore/src/utilities/units/Scale.cpp

```diff
@@ -114,6 +114,20 @@ namespace openstudio{
     static Scale M = {"M", "mega",6,1.0E6};
     return M;
   }
+  
+  /** Static constant defining SI prefix for 10^5. */
+  UTILITIES_API const Scale& hectokilo()
+  {
+    static Scale hk = {"_hk", "hectokilo",5,1.0E5};
+    return hk;
+  }
+
+  /** Static constant defining SI prefix for 10^4. */
```

Comment:
That's the thing though, not having a named scale for 1E-4 was the bug.  When you do cm^2 you get centi = 1E-2 raised to the 2 power = 1E-4.  When the ScaleFactory looked up this scale it didn't find anything so it returned the next closest scale (milli) plus a fudge factor.  The fudge factor got dropped because there was no place for it to get added and you ended up with the wrong conversion.

I added a couple of other scales to prevent this from happening in other situations.